openapi/bisstats/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to BIS statistical data and metadata from Ballerina
+
+### Package Overview
+The `ballerinax/bisstats` is a [Ballerina](https://ballerina.io/) connector for BIS statistical data and metadata.
+This package provides the capability to easily access BIS statistical data and metadata.
+
+#### Compatibility
+|                               | Version               |
+|-------------------------------|-----------------------|
+| Ballerina Language Version    | **Swan Lake Alpha 5** |
+| BIS SDMX Restful API Version  | **v1**                |
```

Comment:
I guess we should follow this format
```
| Ballerina Language Versions  |      BIS SDMX Restful API        |
|:----------------------------:|:--------------------------------:|
|       Swan Lake Alpha 5      |                 V1               |
```

Reply:
Resolved & amend to same commit. Thank you.